docs/HowTo/centos8-eol-solution.md

```diff
@@ -17,20 +17,20 @@ It is worth mentioning that both CentOS and RHEL and EuroLinux are systems
 built on the same source code, so they provide the same functionality. They
 differ mainly in branding.
 
-The operation of switching support is simple, safe and completely reversible.
-What is very important, it requires neither reinstallation of the system nor
-the applications installed on it. The process consists of switching the
-repository, installing the el-release package and updating the system. After
-the switch, CentOS will still be usable, even in production environments. 
+The operation of switching support is simple, reasonably safe and completely
+reversible.  What is very important, it requires neither reinstallation of the
+system nor the applications installed on it. The process consists of switching
+the repository, installing the el-release package and updating the system.
```

Comment:
And reinstalling the system packages inplace?

Reply:
Changed